### ISSUE TITLE
IEA with 7.1.2.2

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#3558]
P22068

GO:0046961 | proton-transporting ATPase activity, rotational mechanism | IEA with 7.1.2.2 |   | GO_REF:0000003

is ATPsynthase subunit not ATPase. This might mean that 7.1.2.2 can't be mapped to GO

**Comment by pgarmiri** [comment]
Hi  Val,
There is a experimental annotation from Pombase for the ATPase function as well. This is based on PMID: 1831760 (Beta subunit of mitochondrial F1-ATPase from the fission yeast. Deduced sequence of the wild type protein and identification of a mutation that increases nucleotide binding.)

![image](https://user-images.githubusercontent.com/22025005/102065091-05949100-3df0-11eb-9ccd-70e95fd1a64a.png)


The electronic mapping with specific EC number is to both terms
![image](https://user-images.githubusercontent.com/22025005/102065660-c3b81a80-3df0-11eb-9361-9b39b9ba0ec2.png)

Could they be both correct?

Thanks,

Penelope

**Comment by ValWood** [comment]
No, that was incorrect, I also just removed that IMP annotation. The F1-ATPase in Eukaryotes only functions in the ATP synthase direction.

**Comment by pgarmiri** [comment]
okay. So, it is a taxon issue..  Could it be solved with a taxon constrain for Eukaryotes?

**Comment by pgaudet** [comment]
> The F1-ATPase in Eukaryotes only functions in the ATP synthase direction.

OK , but the V-ATPase functions as an ATPase. I think this term covers both ? See comment in 
https://enzyme.expasy.org/EC/7.1.2.2

> The V-type (in vacuoles and clathrin-coated vesicles) and A-type (archaeal) enzymes have a similar structure but, under physiological conditions, they pump H(+) rather than synthesize ATP.

**Comment by ValWood** [comment]
Yes that is true, if I understand correctly the EC covers both, ATPase and ATP synthase, but the mapping won't always be true so it will need to be removed? (the mapping needs to be true for everything with that EC number)

**Comment by pgaudet** [comment]
That sounds right. 

We have this mapping on 2 terms: 

GO:0046933
name: proton-transporting ATP synthase activity, rotational mechanism
namespace: molecular_function
def: "Enables the synthesis of ATP from ADP and phosphate by the transfer of protons from one side of a membrane to the other by a rotational mechanism driven by a gradient according to the reaction: ADP + H2O + phosphate + H+(in) = ATP + H+(out)." [RHEA:57722]
synonym: "H+-transporting ATP synthase activity" RELATED [EC:7.1.2.2]
synonym: "hydrogen ion translocating F-type ATPase activity" EXACT []
synonym: "hydrogen ion transporting ATP synthase activity, rotational mechanism" EXACT []
synonym: "hydrogen ion transporting two-sector ATPase activity" EXACT []
xref: EC:7.1.2.2
xref: MetaCyc:ATPSYN-RXN
xref: RHEA:57722
xref: TC:3.A.2.1.1

and 

[Term]
id: GO:0046961
name: proton-transporting ATPase activity, rotational mechanism
namespace: molecular_function
def: "Enables the transfer of protons from one side of a membrane to the other according to the reaction: ATP + H2O + H+(in) = ADP + phosphate + H+(out), by a rotational mechanism." [RHEA:57721]
synonym: "ATP phosphohydrolase (H+-transporting) activity" RELATED []
synonym: "ATP synthase activity" BROAD [EC:7.1.2.2]
synonym: "chloroplast ATPase activity" NARROW [EC:7.1.2.2]
synonym: "F(0)F(1)-ATPase activity" NARROW [EC:7.1.2.2]
synonym: "F(1)-ATPase activity" NARROW [EC:7.1.2.2]
synonym: "F(o)F(1)-ATPase activity" NARROW [EC:7.1.2.2]
synonym: "F0F1-ATPase" NARROW [EC:7.1.2.2]
synonym: "F1-ATPase" NARROW []
synonym: "FoF1-ATPase" NARROW [EC:7.1.2.2]
synonym: "H(+)-transporting ATP synthase activity" RELATED [EC:7.1.2.2]
synonym: "H+-transporting ATPase activity" RELATED [EC:7.1.2.2]
synonym: "hydrogen ion translocating A-type ATPase activity" NARROW []
synonym: "hydrogen ion translocating F-type ATPase activity" NARROW []
synonym: "hydrogen ion translocating V-type ATPase activity" NARROW []
synonym: "hydrogen ion transporting ATPase activity, rotational mechanism" EXACT []
synonym: "hydrogen ion transporting two-sector ATPase activity" NARROW []
synonym: "mitochondrial ATPase activity" NARROW [EC:7.1.2.2]
xref: EC:7.1.2.2
xref: MetaCyc:ATPSYN-RXN
xref: RHEA:57721

* RHEAs do not link to ECs, which is consistent with EC:7.1.2.2 not being mappable to a single reaction
* MetaCyc:ATPSYN-RXN describes GO:0046933 so it should be removed from GO:0046961


@balhoff Can you show me how to manually add NARROW xrefs ? 

Thanks, Pascale
